### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by jennur** [inveniosoftware/invenio-app-rdm#1362]
The color contrast is not sufficient on some elements (global issue). 

__Useful tools__
- [axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=no) to identify contrast issues.
- [Webaim color contrast checker](https://webaim.org/resources/contrastchecker/). The contrast should at least satisfy the WCAG AA requirement.

Some examples:
<img width="286" alt="Screenshot 2022-03-04 at 09 53 23" src="https://user-images.githubusercontent.com/21052053/157204562-c125cebd-88cc-49e1-959b-0dd435907aef.png">
<img width="271" alt="Screenshot 2022-03-04 at 09 53 12" src="https://user-images.githubusercontent.com/21052053/157204576-11557b53-2d82-40d7-b488-e2db3bf12b25.png">

**Comment by amccollo** [comment]
##More information:

- The visual presentation of text and images of text should have a contrast ratio of at least 4.5:1

If needed, I can supply hex codes for the closest color matches that are compliant (email: aaron.mccollough@ubiquitypress.com). 


###On search results pages:

* Number of results found = 3.76:1 against gray background
* Number of results in facet lists = 3.76:1 against gray background
* Deposit date label (White lettering on blue field) = 4.06:1
* Resource type label (white lettering on gray field) = 4.03:1
* Metadata-only label (while lettering on teal field) = 2.56:1 

###On upload form:

* invenio-accordion-field dividers (white lettering on blue background) = 4.06:1
* Storage available labels (gray text on gray field) = 3.76:1
* default field text prompts (gray on white background) = 1.68:1

###On community setup form:

* invenio-accordion-field dividers (white lettering on blue background) = 4.06:1
* default field text prompts (gray on white background) = 1.68:1


###On published record page:

* Technical metadata (gray text on gray background) = 4.44:1
* Resource type label (white lettering on gray field) = 4.03:1
* Metadata-only label (while lettering on teal field) = 2.56:1
